docs/emitters/json-schema/reference/index.mdx

```diff
@@ -31,29 +31,29 @@ npm install --save-peer @typespec/json-schema
 
 ## Emitter usage
 
-[See documentation](./emitter.md)
+[See documentation](./emitter/)
 
 ## TypeSpec.JsonSchema
 
 ### Decorators
 
-- [`@baseUri`](./decorators.md#@TypeSpec.JsonSchema.baseUri)
-- [`@contains`](./decorators.md#@TypeSpec.JsonSchema.contains)
-- [`@contentEncoding`](./decorators.md#@TypeSpec.JsonSchema.contentEncoding)
-- [`@contentMediaType`](./decorators.md#@TypeSpec.JsonSchema.contentMediaType)
-- [`@contentSchema`](./decorators.md#@TypeSpec.JsonSchema.contentSchema)
-- [`@extension`](./decorators.md#@TypeSpec.JsonSchema.extension)
-- [`@id`](./decorators.md#@TypeSpec.JsonSchema.id)
-- [`@jsonSchema`](./decorators.md#@TypeSpec.JsonSchema.jsonSchema)
-- [`@maxContains`](./decorators.md#@TypeSpec.JsonSchema.maxContains)
-- [`@maxProperties`](./decorators.md#@TypeSpec.JsonSchema.maxProperties)
-- [`@minContains`](./decorators.md#@TypeSpec.JsonSchema.minContains)
-- [`@minProperties`](./decorators.md#@TypeSpec.JsonSchema.minProperties)
-- [`@multipleOf`](./decorators.md#@TypeSpec.JsonSchema.multipleOf)
-- [`@oneOf`](./decorators.md#@TypeSpec.JsonSchema.oneOf)
-- [`@prefixItems`](./decorators.md#@TypeSpec.JsonSchema.prefixItems)
-- [`@uniqueItems`](./decorators.md#@TypeSpec.JsonSchema.uniqueItems)
+- [`@baseUri`](./decorators/#@TypeSpec.JsonSchema.baseUri)
```

Comment:
This is a bit unfortunate but astro doesn't rewrite the `.md` links

Reply:
😢

Reply:
reverted that change for now and used a plugin as it makes the docusuarus website unhappy. Seeing how there isn't a large amount of people complaining about this I think its ok that we consider changing the links when we remove the docusaurus build